### ISSUE TITLE
Foundry does not launch any worlds using SWSE system

### ISSUE THREAD

**Issue by SuperSpirals** [kypvalanx/Foundry-VTT-StarWars-SagaEdition#33]
Currently using version 0.3.2
When you attempt to launch the game it gives the following message: "you must define a valid Item type"

After you click the launch button, it also affects your ability to launch other worlds, or the ability to launch updates. Clicking these buttons give the error: "you lack Server Administrator permission to submit this request". Closing and restarting Foundry fixes this.

**Comment by SuperSpirals** [comment]
![image](https://user-images.githubusercontent.com/82356019/119525092-3936f800-bd4c-11eb-8426-49d1892f94e2.png)

**Comment by kypvalanx** [comment]
at the moment, first boot has to be done with full GM privs.  this will be fixed soon.  if you already are using those privs, i'm not sure what's wrong and i'll have to look into it more.

**Comment by SuperSpirals** [comment]
Yeah, i can boot up a new world and launch immediately. the only issue is launching the world i made when you first launched.

I did a workaround by creating a new world, launching it, copying everything in the world's Packs folder, and then pasting them into the old world. In doing so, i found there is a pack in the old world that is not present in the new one, named "swse-abilities". I deleted this file, and then went back and was able to launch the world. Perhaps that extra file was the issue? just speculation. When my friend (who has the exact same issue, cant launch his original world due to this error, but can launch newly created worlds) has free time, I will attempt to simply delete that extra file and see if that allows him to launch his old world again.
